pl/firefox-ios.xliff

```diff
@@ -2863,7 +2863,7 @@
   <body>
    <trans-unit id="Bookmarks Menu">
     <source>Bookmarks Menu</source>
-    <target>Zakładki</target>
+    <target>Menu Zakładki</target>
```

Comment:
It says to match bookmarks.folder.menu.label on Android, and there it’s one of the folders under “Zakładki”: Pasek zakładek, Menu Zakładki and Pozostałe zakładki.

So using just Zakładki instead would make it a “Zakładki” submenu under the “Zakładki” menu, which is highly confusing to use.